src/Nethermind/Chains/chiado.json

```diff
@@ -43,6 +43,8 @@
     "networkID": 10200,
     "gasLimitBoundDivisor": "0x400",
     "maximumExtraDataSize": "0x20",
+    "maxCodeSize": "0x6000",
+    "maxCodeSizeTransition": "0x0",
```

Comment:
It should be Shanghai Transition timestamp, right?

Reply:
Instead of `maxCodeSizeTransition` should be `maxCodeSizeTransitionTimestamp` with the correct value.